src/js/sw/sw.js

```diff
@@ -217,3 +220,53 @@ const fetchHandler = async (event) => {
 self.addEventListener('install', precacheAssets);
 self.addEventListener('activate', clearOldCaches);
 self.addEventListener('fetch', fetchHandler);
+
```

Comment:
Shall we gate all the code added only `if (!('BackgroundFetchManager' in self))`?

Reply:
@beaufortfrancois Thank you, addressed in 7979e32.